src/assets/discuss/html/footer.html

```diff
@@ -88,7 +88,7 @@ <h5 class="footer-h5"><a href="https://layer5.io/company/contact">Contact</a></h
         <div class="footer-bottom row">
             <hr class="footer-hr" />
             <div class="footer-bottom-left">
-                <p>2021 Copyright ©Layer5, Inc | All Rights Reserved</p>
+                <p>2022 Copyright ©Layer5, Inc | All Rights Reserved</p>
```

Comment:
@Ayush7614 You can easily make it dynamic like this.
<script>document.write(new Date().getFullYear())</script>

Reply:
@Ayush7614 updates? you can directly use @YashKamboj 's approach to implement this

Reply:
@Ayush7614 is this something that you can do?